mockDB.js

```diff
@@ -0,0 +1,103 @@
+var config = require('./config.js'),
+    crudService = require('./scripts/crud-service.js'),
+    doc = require('./scripts/doc-config.js'),
+    usage = require('./scripts/usage.js');
+
```

Comment:
Use config folder, so you don't have one giant config object.  config.db.mongo.url.... is a bad sign.  Should just be an object 1 level deep.